pkg/releasenotes/releasenotes.go

```diff
@@ -137,6 +138,9 @@ func (c *Client) Get(org, repo, branch, milestone string) (ReleaseNotes, *Statis
 				Author:      fmt.Sprintf("@%s", p.GetUser().GetLogin()),
 				AuthorURL:   p.GetUser().GetHTMLURL(),
 			}
+			if matches[4] == "!" {
+				rn.Typology = "BREAKING CHANGE"
```

Comment:
If `!` exists, this is a BREAKING CHANGE.

Reply:
Correct!